src/components/atomic/poster.tsx

```diff
@@ -0,0 +1,38 @@
+import styled from "styled-components";
+
+const PosterContainer = styled.div`
+	align-items: center;
+	display: flex;
+	flex-direction: column;
+	height: 100%;
+	justify-content: center;
+`;
+
+const PosterStyle = styled.img<{ $borderRadius?: string }>`
+	${({ $borderRadius }) => $borderRadius && `border-radius: ${$borderRadius};`}
+	max-height: 100%;
+	max-width: 100%;
+	object-fit: contain;
+`;
+
+interface PosterProps {
+	alt: string;
+	borderRadius?: string;
+	src?: string;
+}
+
+export const Poster: React.FC<PosterProps> = ({ src, alt, borderRadius }) => {
```

Comment:
Do add a default value, rather than manually doing:
```ts
if (!borderRadius) {
  borderRadius = "1rem";
}
```

What you can do is change the props to this:
```suggestion
export const Poster: React.FC<PosterProps> = ({ src, alt, borderRadius = "1rem" }) => {
```